packages/sdk/src/components/sdk-get/sdk-get.tsx

```diff
@@ -35,7 +35,7 @@ type OwnProps = {
   ) => boolean;
   onSuccess?: (result: SuccessResult['body']) => void;
   onError?: (error: TSdkError) => void;
-  render: (options: TRenderOptions) => JSX.Element;
+  render: (options: TRenderOptions) => React.JSX.Element;
```

Comment:
Since we're already importing react in this file, I think it would be more convenient to import the `JSX` namespace instead of relying on the global React object.